test/e2e/features_test.go

```diff
@@ -0,0 +1,334 @@
+//go:build e2e_tests
+// +build e2e_tests
+
+package e2e
+
+import (
+	"context"
+	"crypto/tls"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/blang/semver/v4"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/loadimage"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/addons/metallb"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/types/kind"
+	"github.com/kong/kubernetes-testing-framework/pkg/environments"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// -----------------------------------------------------------------------------
+// E2E feature tests
+//
+// These tests test features that are not easily testable using integration
+// tests due to environment requirements (e.g. needing to mount volumes) or
+// conflicts with the integration configuration.
```

Comment:
```suggestion
// have conflicts with the integration configuration.
```

Reply:
Nah, both sides of the or are noun phrases, split is after "due to"